h/templates/app.html

```diff
@@ -16,6 +16,7 @@
           --><span class="provider" ng-show="auth.user">/{% raw %}{{ auth.user|persona:'provider' }}{% endraw %}</span><!--
           --><i class="h-icon-arrow-drop-down"></i></span>
         <ul class="dropdown-menu pull-right" role="menu">
+          <li ng-hide="isStream"><a href="" ng-click="viaLinkVisible = true">Share this page</a></li>
```

Comment:
If we have a share button in the search bar, let's maybe not have it in the dropdown.